src/porepy/examples/flow_benchmark_2d_case_1.py

```diff
@@ -12,43 +12,36 @@
 
 """
 
-from typing import Callable, Union
+from dataclasses import dataclass
+from typing import Callable, ClassVar, Union
 
 import numpy as np
 
 import porepy as pp
 from porepy.applications.discretizations.flux_discretization import FluxDiscretization
+from porepy.compositional.materials import SolidConstants, _HashableDict
```

Comment:
Does the import suggest that the class shouldn't be private?

Reply:
If so, the location in compositional.materials is also strange. 

If I understand correctly, the import is needed to allow typing of extensions of the material constants? If yes, I believe this is something we want to allow, so should we find a better home for it?

Reply:
I am open for any change in location for better fit.

Reply:
With the latest changes, there is no need for a hashable dict and I removed that class.
Can we resolve this?